pages/views.py

```
@@ -0,0 +1,36 @@
+                if x + y > 20:
+                    score = x * y
+                else:
+                    if y == -1:
+                        score = 0
+            else:
+                if y > 10:
+                    score = x + y
+        else:
+            if x == 0:
+                score = y
+        return HttpResponse(f'Special score: {score}')
```

Pylint (3.3.7)

[error] Using variable 'score' before assignment

(E0601)